rero_ils/static/scss/rero_ils/variables.scss

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2015-2018 CERN.
+ * Copyright (C) 2018 RERO.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License;
+ * See license file for more details.
+ */
+
+$navbar-height: 60px;
+$brand-font-family: "Helvetica Neue", Helvetica, Arial, sans-serif !default;
+$brand-font-size: 30px;
```

Comment:
Why not em ?

Reply:
@Garfield-fr These variables are set by `invenio-theme`, it's just a copy'n'paste...